src/main/java/roomescape/domain/ReservationTimeAvailabilities.java

```diff
@@ -0,0 +1,31 @@
+package roomescape.domain;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ReservationTimeAvailabilities {
+
+    private final Map<ReservationTime, Boolean> values;
+
+    public static ReservationTimeAvailabilities of(final List<ReservationTime> reservationTimes, final List<Reservation> reservations) {
+        final Map<ReservationTime, Boolean> reservationTimeAvailabilities = new HashMap<>();
+        reservationTimes.forEach(reservationTime -> reservationTimeAvailabilities.put(
+                reservationTime, isTimeAvailable(reservations, reservationTime)));
```

Comment:
그냥 for 문을 사용하고 줄바꿈을 적절하게 해서 가독성을 높여볼 수 있을 것 같아요:
```suggestion
        for (ReservationTime reservationTime : reservationTimes) {
            reservationTimeAvailabilities.put(
                reservationTime,
                isTimeAvailable(reservations, reservationTime)
            );
        }
```

Reply:
for 문이 더 가독성이 좋을 때가 있군요 😂 
수정해보았습니다